CMakeLists.txt

```diff
@@ -201,6 +204,10 @@ set(priv_includes cores/esp32/libb64)
 set(requires spi_flash mbedtls mdns esp_adc_cal wifi_provisioning nghttp)
 set(priv_requires fatfs nvs_flash app_update spiffs bootloader_support openssl bt esp_ipc esp_hid)
 
+if(NOT CONFIG_ARDUINO_SELECTIVE_COMPILATION OR CONFIG_ARDUINO_SELECTIVE_BluetoothSerial)
```

Comment:
this should be done in the source files and not here

Reply:
Okay ! Wasn't sure which approach was the best, I'll change it :)